addon/routes/dashboard/week.js

```diff
@@ -0,0 +1,3 @@
+import Route from '@ember/routing/route';
+
+export default class DashboardWeekRoute extends Route {}
```

Comment:
This is the same as not having the file at all, so it can probably be removed.

Reply:
:heavy_check_mark: